assets/stories/skate-components/map/controls/streetViewSwitch.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { argsUpdater } from "../../../../.storybook/argsUpdater"
+
+import { StreetViewSwitch } from "../../../../src/components/map/controls/StreetViewSwitch"
+
+import { Decorator } from "@storybook/react"
+import React from "react"
+
+const inControlDivDecorator: Decorator = (StoryFn) => (
+  <div className="leaflet-control leaflet-bar c-street-view-switch position-absolute">
+    <StoryFn />
+  </div>
+)
+
+const meta = {
+  component: StreetViewSwitch,
+  args: {
+    streetViewEnabled: false,
+  },
+  argTypes: {
+    setStreetViewEnabled: { table: { disable: true } },
+  },
+  decorators: [
+    argsUpdater(
+      "setStreetViewEnabled",
+      ({ streetViewEnabled }, newStreetViewEnabled) => ({
+        streetViewEnabled:
+          typeof newStreetViewEnabled === "function"
+            ? newStreetViewEnabled(streetViewEnabled)
+            : newStreetViewEnabled,
```

Comment:
note(non-blocking): I think we're probably gonna want a utility for this type of thing at some point?
But definitely not something to do right now.